### DISCUSSION TITLE
How to map PostgreSQL bytea to Prisma?

### DISCUSSION THREAD

**Discussion by arthurlomakin11** [prisma/prisma#11976] [Q&A] [answered]
I need to have field with type bytea in PostgreSQL and mapped to Buffer in Node. How can I do this?

**Comment by nurul3101** [comment, marked as answer]
Hey @arthurlomakin11 👋🏻 ,
You could do this by defining your model in the following way:

```prisma
model User {
  id    Int   @id
  field Bytes @db.ByteA
}
```
Here is a reference to working with [Bytes](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#bytes) in prisma schema.

Please let me know if this solves the issue for you.

**Reply by arthurlomakin11** [reply]
@nurul3101 Solved the problem, thank you!

**Reply by hammadkhokhar** [reply]
Thanks mate!

**Reply by brandao07** [reply]
Thank you !

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
